concourse/content-event-resource/src/in_.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+import sys
+
+from src import event_api as api
+from src.utils import msg
+
+
+def in_(dest_path, in_stream):
+    input = json.load(in_stream)
+    api_root = input["source"]["api_root"]
+    event_id = input["version"]["id"]
+
+    event = api.get_event(api_root, event_id)
+    msg("Event Returned: {}", event["id"])
+
+    # Write out files
+    with open(os.path.join(dest_path, "id"), "w") as file:
+        file.write(str(event_id))
+
+    with open(os.path.join(dest_path, "ident_hash"), "w") as file:
+        file.write(event["ident_hash"])
```

Comment:
FYI This is our input to the bridge resource. It'll be a parameter in `get` to something like `ident_hash_file`.

Reply:
Do you want me to rename it?